### PR TITLE
Hide workload list options only relevant to machine readable

### PR DESCRIPTION
**Pull request by sfoslund**

### CODE REVIEW

File: src/Cli/dotnet/OptionForwardingExtensions.cs [hunk 1]
```diff
@@ -44,6 +44,12 @@ public static Option<T> AllowSingleArgPerToken<T>(this Option<T> option)
             return option;
         }
 
+        public static Option<T> MakeHidden<T>(this Option<T> option)
```
Comment (joeloff) [inline src/Cli/dotnet/OptionForwardingExtensions.cs:47]: Change looks good, wondering if we should call this Hide instead

Comment (sfoslund) [reply src/Cli/dotnet/OptionForwardingExtensions.cs]: Sure, just renamed.

**Comment by sfoslund** [comment]
@joeloff anything else to do here or is it good to go?

**Review by joeloff — APPROVED** [review]
LGTM, thanks

**Pull request merged**